docs/app/pages/components/components-sections/tables/column-picker/snippets/app.ts

```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
-import { ColumnPickerGroup, ColumnPickerValue } from '@ux-aspects/ux-aspects';
+import { ColumnPickerGroup } from '@ux-aspects/ux-aspects';
+import { ColumnPickerGroupItem } from '@ux-aspects/ux-aspects/components/table/column-picker/interfaces/column-picker-group-item.interface';
```

Comment:
Import this from @ux-aspects/ux-aspects, just like you would in an application (see the line above).